manifests/params.pp

```diff
@@ -98,7 +92,7 @@
       }
     }
     'RedHat': {
-      $config_root_ini         = '/etc/php.d'
+      $config_root             = $php::globals::globals_config_root
```

Comment:
this is the reason rhel fails. `$config_root_ini` should be the same as `$config_root`, if i see this correctly